website/docs/docs/collaborate/govern/project-dependencies.md

```diff
@@ -0,0 +1,30 @@
+---
+title: "Project dependencies"
+id: project-dependencies
+sidebar_label: "Project dependencies"
+description: "Reference public models across dbt projects"
+---
+
+:::info Cross-project `ref`
+This is a feature of dbt Cloud Enterprise.
+:::
+
+dbt Core v1.6 introduces a notion of `dependencies` between dbt projects. You might already be familiar with installing other projects as [packages](/docs/build/packages), whereby you pull down another project's source code and treat it as your own.
+
+There is a new kind of `project` dependency. Both dependencies can be defined in `dependencies.yml`:
```

Comment:
It feels like there is gap between the opening sentences and this one. Is there some more information we want to share to help users connect the dots here?

Reply:
writing more here!